python/GafferUI/_StyleSheet.py

```diff
@@ -42,37 +42,71 @@
 
 _styleColors = {
 
+	# The theme is build on the principal of using color variation and
+	# subtle embossing instead of lines to differentiate UI elements.
+	# There a few holes in this, but it hopefully results in a lower overall
```

Comment:
There -> There are

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3229/commits/1812b016aa67d37345646a5ed5d16dca5918826e